Makefile

```diff
@@ -0,0 +1,54 @@
+SLEEP = 60
+DOCKER_COMPOSE_DEV = docker-compose-dev.yml
+DOCKER_TEST = docker exec -i anodot-agent pytest
+
+all: build test-all
+
+build:
+	docker-compose up -d --build
+	sleep $(SLEEP)
+	./setup_tests.sh
+
+test-all:
+	$(DOCKER_TEST) -x
+
+all-dev: build-dev test-all-dev
+
+build-dev:
+	docker-compose -f $(DOCKER_COMPOSE_DEV) up -d --build
+	docker exec -i anodot-agent python setup.py install
```

Comment:
for the dev image you need to execute `python setup.py develop` instead of `python setup.py install`